src/patient-search/patient-search-result/patient-search-result.component.css

```diff
@@ -0,0 +1,79 @@
+@media screen and (min-width: 769px) {
+  .patientResult {
+    display: flex;
+    justify-content: center;
+  }
+  .resultNumber {
+    display: block;
+    margin: 0rem 0.75rem;
+    color: var(--omrs-color-ink-medium-contrast);
+    padding: 1rem;
+  }
+  .patientCard {
+    border-radius: 0.75rem;
+  }
+  .tile {
+    padding: 0.625rem 1.5rem;
+  }
+}
+
+@media screen and (max-width: 769px) {
+  .resultNumber {
+    display: none;
+  }
+  .tile {
+    padding: 0.625rem 1rem;
+  }
+}
+.patientResult {
+  margin-top: 0.75rem;
+}
+.patientDetailsContainer {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.patientNameContainer {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 1rem;
+}
+
+.patientName {
+  font-size: 22px;
+  line-height: 129%;
+  letter-spacing: 0.03em;
+  font-weight: bold;
+  color: var(--omrs-color-interaction);
+}
+
+.patientData {
+  font-size: 1.1875rem;
```

Comment:
same here -- use the typography css classes instead of recreating.

Reply:
I checked, there's no typography for this style